mac/Info-make.plist

```diff
@@ -21,7 +21,7 @@
         <key>LSApplicationCategoryType</key>
 	    <string>public.app-category.music</string>
         <key>LSMinimumSystemVersion</key>
-        <string>10.13.0</string>
+        <string>${MACOSX_DEPLOYMENT_TARGET}</string>
```

Comment:
Yes, this is indeed supposed to be `${MACOSX_DEPLOYMENT_TARGET}`, not `@MACOSX_DEPLOYMENT_TARGET@` due to the `sed` command generated in the `Makefile`.